### PR TITLE
Update ABOUT.md to nextercism format

### PR DESCRIPTION
**Pull request by lindakatcodes**
As requested in issue [#85](https://github.com/exercism/meta/issues/85) in the meta repo - some minor formatting changes, along with a few grammatical/wording fixes.

### CODE REVIEW

**Comment by iHiD** [comment]
This looks good to me - thank you :) @kytrinyx Any comments?

File: docs/ABOUT.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+Server-Side JavaScript as run in NodeJS enables back-end access to databases, file systems, and servers. NodeJS is built on Google Chrome's JavaScript V8 Engine. NodeJS uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node can be a great solution for applications requiring I/O bound operations, data streaming etc. More details can be found [here](https://nodejs.org/en/about/).
 
 You should learn JavaScript because:
 
 * It's easy to learn.
-* It's versatile in the sense that it's multi-paradigm supporting procedural, event based, object oriented and functional programming.
-* It can be used for the frontend and backend.
+* It's versatile in the sense that it's multi-paradigm - supporting procedural, event based, object oriented and functional programming.
```
Comment (matthewmorgan) [inline docs/ABOUT.md]: Should read:
> It's versatile in the sense that its multi-paradigm

File: docs/ABOUT.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
-Javascript is a scripting language used to provide dynamic and interactive content on webpages. Also, server side JS allows the use of the same language on the server and client. Besides being fast, JavaScript provides benefits like:
+JavaScript is a scripting language used to provide dynamic and interactive content on webpages. Also, server side JS allows the use of the same language on the server and client.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Would you mind starting the second sentence on a new line?

File: docs/ABOUT.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 * Allowing richer user interfaces with content changes on mouse hover, drag and drop gestures, and animations.
 
 Client-side JavaScript is interpreted in the browser without requiring compilation. This allows interactive content to be included in HTML pages which would otherwise be static.
 
-Server-Side JavaScript as run in NodeJS enables back-end access to databases, file systems, and servers. NodeJS is built on Google Chrome's JavaScript V8 Engine. NodeJS uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node can be a great solution for applications requiring I/O bound operations, data streaming etc. More details can be found  [here](https://nodejs.org/en/about/).
+Server-Side JavaScript as run in NodeJS enables back-end access to databases, file systems, and servers. NodeJS is built on Google Chrome's JavaScript V8 Engine. NodeJS uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node can be a great solution for applications requiring I/O bound operations, data streaming etc. More details can be found [here](https://nodejs.org/en/about/).
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Here as well, we'd like to have each sentence on its own line.

File: docs/ABOUT.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
-* It can be used for the frontend and backend.
+* It's versatile in the sense that it's multi-paradigm - supporting procedural, event based, object oriented and functional programming.
+* It can be used for the front-end and back-end.
 * It's Open Source.
 * JavaScript programming skills are in high demand.
 
----
-
-_This track is intended for widely supported JavaScript, for code that could be executed in almost all existing browsers. If you're looking for experimenting with newer features of the language try the [ECMAScript](http://exercism.io/languages/ecmascript) track._
+_This track is intended for widely supported JavaScript, for code that could be executed in almost all existing browsers. If you're looking to experiment with newer features of the language, try the [ECMAScript](http://exercism.io/languages/ecmascript) track._
```
Comment (kytrinyx) [inline docs/ABOUT.md]: And here as well :-)

**Comment by lindakatcodes** [comment]
Alright, I've adjusted the file to have each sentence on it's own line. It looked the same in the preview to me, but the code file had them adjusted, so I'm hoping that works as requested? If not let me know and I'll keep tweaking it. :)

**Pull request merged**